### DISCUSSION TITLE
TypeScript declaration packages for Parcel

### DISCUSSION THREAD

**Discussion by 101arrowz** [parcel-bundler/parcel#5060] [General] [answered]
Just linking DefinitelyTyped/DefinitelyTyped#46996. My attempt at adding TypeScript types to Parcel - I don't know if there's anything that I thought was internal but was actually meant for external use, so feedback is appreciated.

**Comment by mischnic** [comment]
-> https://github.com/parcel-bundler/parcel/issues/3912#issuecomment-678957791

**Comment by 101arrowz** [comment, marked as answer]
#5297
